weaver/processes/wps_process_base.py

```diff
@@ -465,19 +471,41 @@ def dispatch(self, process_inputs, process_outputs):
             "response": ExecuteResponse.DOCUMENT,
             "inputs": process_inputs,
         }
+        execute_headers = {
+            "Prefer": "respond-async",
+            "Accept": ContentType.APP_JSON,
+            "Content-Type": ContentType.APP_JSON,
+        }
```

Comment:
This header will never be updated ?

Reply:
Not in this case because the pipeline always expect to receive the "document" response to extract all the results available from the job in the same matter, whether there is only one or many.